README.md

```diff
@@ -24,7 +24,7 @@ But you won't need to worry about installing any of these - they are automatical
 
 ## How to run
 
-If you really need to run DeviceManager as a standalone process (without dojot's wonderful docker-compose), you can execute these commands:
+If you really need to run DeviceManager as a standalone process (without dojot's wonderful [docker-compose](https://github.com/dojot/docker-compose)), you can execute these commands:
```

Comment:
I think that a note of warning related to user management is due here.
That should be specially important if we transition to a more strict checking of tokens on services (i.e. make proper use of the "dummy" strings at the beginning and end of the token)

Reply:
I'll add a comment on that.